sourcegraph/sourcegraph
# search ui: permanently enable simple ui (but keep temp setting)

Closes https://github.com/sourcegraph/sourcegraph/issues/41001

- [x]  Set the Simple UI feature flag to be on by default.
- [x]  Remove the toggle, keep the temporary setting.
- [x]  Flip search history feature flag, so that it is off by default and the feature flag enables it.

## Test plan

- Chromatic Storybook has been updated so that all stories have Simple UI enabled. Some irrelevant stories (eg. homepage panels) have been removed, and some new stories have been added to test the new UI (eg. server home with chips).
- Updated unit tests to be based on the new UI.
- Verify updated Storybook and Percy snapshots.
- Verify that users without any temp settings get the new UI by [clearing the temporary settings](https://docs.sourcegraph.com/dev/background-information/web/temporary_settings#authenticated-users).
- Verify that users who have previously turned off Simple UI via toggle get the new UI anyway.
- Verify that the search history is off by default and only shown if feature flag `search-input-show-history` is enabled.

## App preview:

- [Web](https://sg-web-jp-enablesimpleui.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-tvmuhthcud.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
